inst/rmarkdown/templates/pdf/skeleton/pinp.cls

```diff
@@ -206,14 +222,18 @@
 \RequirePackage[noend]{algpseudocode}
 \RequirePackage{changepage}
 \RequirePackage[twoside,%
-				includeheadfoot,
+				letterpaper,includeheadfoot,%
+				layoutsize={8.125in,10.875in},%
```

Comment:
From what I recall, this was dropped in the `pinp` class before.

Reply:
As in not enforcing us paper?  Possibly.  

(Check `master`)

Yup.  So just nix the line? Or is there more?  Can you find the commit(s) where we did that?  (Just in from a run myself...)